### ISSUE TITLE
YouTube livestreams stop playing

### ISSUE THREAD

**Issue by yokoffing** [yokoffing/filterlists#160]
### Prerequisites

- [X] I am not using any other content blockers alongside uBO (Privacy Badger, AdGuard, Ghostery, Adblock Plus, etc.)
- [X] I verified that this is not a [known filter issue](https://github.com/yokoffing/filterlists/issues)
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the uBO issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBO in the browser

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Specific URL(s) where the issue occurs

[YouTube livestreams](https://www.youtube.com/channel/UC4R8DWoMoI7CAwX8_LjQHig)

### Description

Migrating from https://github.com/yokoffing/Betterfox/issues/323

### Logger Output

```shell
see previous issue
```

### Expected behavior

Live streams to work

### Actual behavior

Live streams stop playing with Privacy Essentials list.

**Comment by yokoffing** [comment]
@gaboscharff99 I actually think the issue lies in `yokoffing's Annoyance List`, but I am testing filters one-by-one.

**Comment by gaboscharff99** [comment]
I'm fairly certain I've tested with the Annoyance List disabled and with Privacy Essentials enabled, and it didn't work either. Let me try again and I'll let you know.

**Comment by gaboscharff99** [comment]
Well, this is odd. Now I can't reproduce the issue with either list. Is it possible it was a specific combination of lists?

**Comment by gaboscharff99** [comment]
Oddly enough, I can't reproduce the issue with either list now. Is it possible it might be a specific combination of lists?

**Comment by yokoffing** [comment]
Possibly, it was. https://github.com/yokoffing/filterlists/commit/a87a294212f8182a5bf1e05ccd2cb00f83ac1be0 https://github.com/yokoffing/filterlists/commit/3f3d8047dde114412ced2f5ee610fd2484b0d027

**Comment by gaboscharff99** [comment]
I'm glad to report that, after testing with all filterlists found on the Pro example in the readme (except for YouTube Clear View, as I personally don't wish to use it), YouTube livestreams now work as intended. Thank you for looking into this!

**Comment by yokoffing** [comment]
Glad to hear it! Let me know if things change.
